maven-archetype-plugin/src/main/java/org/apache/maven/archetype/ui/ArchetypeConfiguration.java

```diff
@@ -19,12 +19,15 @@
  * under the License.
  */
 
+import org.apache.maven.archetype.common.Constants;
+import org.codehaus.plexus.util.StringUtils;
```

Comment:
Yes, though it was the right thing to do. It is still unrelated to this PR. Code improvements are always welcome and shall be made with separate PRs.

Thank you!